src/block/Potato.php

```diff
@@ -23,23 +23,36 @@
 
 namespace pocketmine\block;
 
+use pocketmine\block\utils\FortuneTrait;
 use pocketmine\item\Item;
 use pocketmine\item\VanillaItems;
 use function mt_rand;
 
 class Potato extends Crops{
+	use FortuneTrait;
 
-	public function getDropsForCompatibleTool(Item $item) : array{
-		$result = [
-			VanillaItems::POTATO()->setCount($this->age >= self::MAX_AGE ? mt_rand(1, 5) : 1)
-		];
+	public function getPickedItem(bool $addUserData = false) : Item{
+		return VanillaItems::POTATO();
+	}
+
+	/**
+	 * @return Item[]
+	 */
+	protected function getFortuneDropsForLevel(int $level) : array{
+		if ($this->age >= self::MAX_AGE) {
+			$result = $this->binomialDrops(
+				VanillaItems::POTATO(),
+				$level,
+				1
+			);
+		} else {
+			$result = [
+				VanillaItems::POTATO()
+			];
+		}
 		if($this->age >= self::MAX_AGE && mt_rand(0, 49) === 0){
 			$result[] = VanillaItems::POISONOUS_POTATO();
```

Comment:
Does fortune have any effect on this?

Reply:
I don't see any information about it in the wiki. Maybe there is, but since we don't have an exact value, I think we can assume not.

Reply:
Everywhere it is quoted that the poisoned potatoes are drop with 2% chance but without more